api/shared.thrift

```diff
@@ -4,6 +4,8 @@
 //   License. See LICENSE.TXT for details.
 // -------------------------------------------------------------------------
 
+const string API_VERSION = '6.0'
```

Comment:
Ok, I've just asked it because there are multiple structs in this file which should be moved to the report server thrift file, if they are used only there.